### ISSUE TITLE
BGE crashes on start: EXCEPTION_ACCESS_VIOLATION

### ISSUE THREAD

**Issue by joelgomes1994** [UPBGE/blender#661]
Hello!

Using latest master, Windows 7 64 bit, MSVSC 2015 32 bit.

Different from #657 , simply by starting the BGE in embedded player or standalone player Blender closes with only the EXCEPTION_ACCESS_VIOLATION in console, in both debug or normal modes. Sometimes it works, but most of the time it doesn't.

![image](https://user-images.githubusercontent.com/19820741/34115195-0785dd58-e3fc-11e7-99a5-1f4a6e4e59ec.png)

**Comment by youle31** [comment]
hi, Thanks for report, I confirm on the bug on windows 32 bits builds (for me the crashes occurs only in release mode but I guess this is the same bug).
We talked about it last time with panzergame. Just to remember, I think this is related to recent changes on RAS_BoundingBox and mathfu

**Comment by panzergame** [comment]
@youle31 : Can you paste a simple backtrace of the crash in release mode ? Else could you check if there's no warning related to allocator ?

**Comment by youle31** [comment]
hi, sorry panzergame I didn't see your message before. It is ok for me in last master. @joelgomes1994 : Can you confirm please? Thanks!

**Comment by joelgomes1994** [comment]
@youle31 
It's working now in last master for me too. Thanks. ;)

**Comment by joelgomes1994** [comment]
@panzergame @youle31 
However I noticed a problem using Python... I tried to print something like:
`print( own )`
And it prints 'Plane'. However, if I change it to:
`print( dir( own ) )`
It prints 'Plane' again, as if the interpreter had not stopped / cleaned the engine data. Only happens in embedded player, it works as expected in standalone player.
Maybe is it related to c83d50e6fce8b2baf934e06f1c9b2300485a5ce2 ?
Should I open a new issue about that?
